<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTask</class>
 <widget class="QDialog" name="CreateTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="createTaskBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите название задачи*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskNameLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите описание задачи*</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="taskDiscPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateStartLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dateStartLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите дату начала выполнения задачи</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dateEndLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>381</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="dateEndLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите дату окончания выполнения задачи</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
